static/app/views/releases/thresholdsList/index.tsx

```diff
@@ -123,10 +136,36 @@ function ReleaseThresholdList({}: Props) {
     return <LoadingIndicator />;
   }
 
+  const createNewThresholdGroup = () => {
+    if (selectedProjects.length === 1) {
+      setNewThresholdGroup([
+        ...newThresholdGroup,
+        {
+          id: `${NEW_GROUP_PREFIX}-${newGroupIterator}`,
```

Comment:
you might be able to just add a unique id instead of tracking how many there have been https://github.com/getsentry/sentry/blob/master/static/app/utils/guid.tsx#L6
```suggestion
          id: `${NEW_GROUP_PREFIX}-${newGroupIterator}`,
```